api/serializers/__init__.py

```diff
@@ -22,6 +22,7 @@
     AssetStatusSerializer,
     AssetSubCategorySerializer,
     AssetTypeSerializer,
+    StateTransitionsSerializer,
```

Comment:
let's change this to `StateTransitionsSerializer`
Ideally the model, view, serializer names should be singular.